Adds an admonition for exclusive drive access (#1196)
minio/docs: source/includes/common-admonitions.rst

.. end-remove-api-trims-prefixes-fs

.. The following exclusive access admonition is used on a number of pages:
   - administration/object-management.rst
   - administration/concepts.rst
   - operations/concepts.rst
   - operations/data-recovery.rst
   - operations/checklists/hardware.rst
   - operations/checklists/software.rst
   - operations/concepts/availability-and-resiliency.rst
   - operations/concepts/erasure-coding.rst
   - operations/data-recover/recover-after-drive-failure.rst
   - operations/data-recover/recover-after-node-failure.rst
   - operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst
   - operations/install-deploy-manage/deploy-minio-single-node-multi-drive.rst
   - operations/install-deploy-manage/deploy-minio-single-node-single-drive.rst
   - operations/install-deploy-manage/deploy-minio-tenant.rst
   - operations/install-deploy-manage/expand-minio-deployment.rst
   - operations/install-deploy-manage/expand-minio-tenant.rst
   - glossary.rst

.. start-exclusive-drive-access

.. admonition:: Exclusive access to drives
   :class: warning

   MinIO **requires** *exclusive* access to the drives or volumes provided for object storage.

>    No other processes, software, scripts, or human interactions should perform any actions directly on the drives or volumes provided to MinIO or the objects or files MinIO places on them.

Reviewer: "human interactions" doesn't read as an entity capable of causing change, maybe "persons" to keep things consistent
```suggestion
   No other processes, software, scripts, or persons should perform any actions directly on the drives or volumes provided to MinIO or the objects or files MinIO places on them.
```

Reviewer: could emphasize _any_ as well

Author: Changed both

>    Unless directed by MinIO Engineering, do not use scripts or non-S3 tools to modify, delete, or move any of the data shards, parity shards, or metadata files on the provided drives, including from one drive or node to another.

Reviewer 2: I'd say any scripts or tools, it's not like an S3 compliant tool should mess with shards either

>    Such operations can easily result in widespread corruption and loss of data beyond MinIO's ability to heal.

Reviewer: a reword alternative, if you like
```suggestion
   Such operations are very likely to result in widespread corruption and data loss beyond MinIO's ability to heal.
```
